lib/core/src/lib/card-view/models/card-view-datetimeitem.model.ts

```diff
@@ -22,7 +22,7 @@ import { CardViewDateItemProperties } from '../interfaces/card-view.interfaces';
 
 export class CardViewDatetimeItemModel extends CardViewDateItemModel implements CardViewItem, DynamicComponentModel {
     type: string = 'datetime';
-    format: string = 'MMM d, y, H:mm';
+    format: string = 'MMM d, y';
```

Comment:
the model file is called "datetime" model, and the "type" is called "datetime", so you can't drop the time from here

Reply:
i have again implemented tim here